lib/index.js

```diff
@@ -14,10 +15,12 @@ module.exports = (rootDir, config = "faucet.config.js", // eslint-disable-next-l
 	let configDir = path.dirname(configPath);
 	config = require(configPath);
 
-	let watcher;
-	if(watch) {
-		watcher = makeWatcher(config.watchDirs, configDir);
-	}
+	let watcher = watch && makeWatcher(config.watchDirs, configDir);
+	let manager = new AssetManager(configDir, {
+		manifestConfig: config.manifest,
+		fingerprint,
```

Comment:
I thought you said that you only use the "compact" version when you can use it for all the keys?

Reply:
```
{
  manifestConfig: x,
  fingerprint: fingerprint,
  bla: y
}
```

vs.

```
{
  manifestConfig: x,
  fingerprint,
  bla: y
}
```

I thought you preferred the first version, and only use the short-cut version if it is applicable to all keys (`{ manifestConfig, fingerprint, bla }`)

Reply:
Oh, right, I thought you were somehow referring to the `compact` option...

I was struggling a little there, but ultimately decided redundantly writing `fingerprint: fingerprint` was kinda silly and no more helpful than using the shorthand. No strong feelings though.

Reply:
IMHO it is harder to read, because it breaks the pattern of the other lines.

Reply:
Thank you. It of course makes sense that you are good at descaling:

![Descaling for professionals](https://www.karinskiosk.de/Bilder/1103210.jpg)

Reply:
How dare you doubt my qualities as a growth hacker?